server/worker.js

```diff
@@ -0,0 +1,22 @@
+const cron = require('node-cron');
+const model = require('./database/queries');
+const Moment = require('moment');
+
+let time;
+let prevJobTime = new Moment().subtract(5, 'years').format('YYYY-MM-DD HH:mm:ss');
```

Comment:
not quite certain what you mean. I set it to 5 years. but the true interval is only 5-10 seconds.